src/frontend/test_runner/tests/testdata/time_window.yaml

```diff
@@ -167,3 +171,12 @@
       StreamHopWindow { time_col: t1.created_at, slide: 1 day 00:00:00, size: 3 days 00:00:00, output: [t1.id, t1.v1, t1.created_at, window_start, window_end, t1._row_id] }
         StreamFilter { predicate: (t1.v1 >= 10:Int32) }
           StreamTableScan { table: t1, columns: [t1.id, t1.v1, t1.created_at, t1._row_id], pk: [t1._row_id], distribution: HashShard(t1._row_id) }
+- sql: |
+    with t(ts) as (values ('2020-01-01 12:00:00'::timestamp)) select * from tumble(t, ts, interval '10' second) as z;
+  logical_plan: |
+    LogicalProject { exprs: [, TumbleStart(, '00:00:10':Interval), (TumbleStart(, '00:00:10':Interval) + '00:00:10':Interval)] }
```

Comment:
I guess this is anonoymous column has no display bug, will be fixed in existing PR.